hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -195,19 +195,19 @@ public final class HddsConfigKeys {
   public static final String HDDS_X509_RENEW_GRACE_DURATION_DEFAULT = "P28D";
 
   public static final String HDDS_EXTERNAL_ROOT_CA_CERT_PATH =
```

Comment:
In order to conform with the convention, I believe the constant names should also be changed here.

Reply:
Thank you!